### PR TITLE
compile mscoffobj.d for all

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Review by ibuclaw — APPROVED** [review]

**Comment by WalterBright** [comment]
Test Suite heisenbug: https://issues.dlang.org/show_bug.cgi?id=21422

File: src/dmd/backend/mscoffobj.d [hunk 1]
```diff
@@ -51,12 +51,12 @@ nothrow:
 alias _compare_fp_t = extern(C) nothrow int function(const void*, const void*);
 extern(C) void qsort(void* base, size_t nmemb, size_t size, _compare_fp_t compar);
 
-static if (TARGET_WINDOS)
+static if (1)
 {
```
Comment (Geod24) [inline src/dmd/backend/mscoffobj.d:55]: Just remove ?

File: src/dmd/backend/mscoffobj.d [hunk 2]
```diff
@@ -1667,7 +1668,7 @@ static if (0) // NOT_DONE
 }
 }
 
-char *unsstr(uint value)
+private extern (D) char *unsstr(uint value)
```
Comment (Geod24) [inline src/dmd/backend/mscoffobj.d:1671]:
```suggestion
private extern (D) char* unsstr(uint value)
```

Since you are changing the line anyway

Comment (WalterBright) [reply src/dmd/backend/mscoffobj.d]: done

**Review by Geod24 — APPROVED** [review]

**Pull request merged**